src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -468,6 +471,51 @@ private void initializeInputPreloadExecutorService() {
         }
     }
 
+    private Map<String, FeatureReader<VariantContext>> createSampleToReaderMap(
+            final Map<String, Path> sampleNameToVcfPath, final int batchSize, final int index) {
+        //TODO: fix casting since it's really ugly
+        return inputPreloadExecutorService != null ?
```

Comment:
In the past we found that the sorting here was really important or it changed the results.  If it's still sensitive to the ordering of the map maybe the signature of `ImportConfig` constructor that this is fed to should be modified to use `SortedMaps`

Reply:
Yes. I think we have kept the same sort order based on the following:
* [The tool sorts by sample name first](https://github.com/francares/gatk/blob/cb46c8acba2a374a2217f27056de5292c4047e5b/src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java#L346)
* [Using the samples in the same order as provided in the sampleToVcfName TreeMap](https://github.com/francares/gatk/blob/cb46c8acba2a374a2217f27056de5292c4047e5b/src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java#L515)
* Passing the same TreeMap as argument to getFeatureReaders()

I did a couple of tests with arbitrary ordering of samples